generators/server/templates/src/main/resources/config/application.yml.ejs

```diff
@@ -195,7 +195,8 @@ spring:
         binder:
           replicationFactor: 1
           auto-create-topics: true
-          brokers: localhost:9092
+          # Add kafka host to your hosts
```

Comment:
I would prefer to avoid: otherwise we would have to do the same for kc, hazelcast, redis, ....
So 10 lines to add to your host before being able to startup your app locally...

Reply:
Testing docker compose.
Fixed the docker env variable.